main/swadge2024.c

```diff
@@ -163,12 +165,19 @@ static swadgeMode_t* cSwadgeMode = &mainMenuMode;
 /// @brief A pending Swadge mode to use after a deep sleep
 static RTC_DATA_ATTR swadgeMode_t* pendingSwadgeMode = NULL;
 
+/// @brief A minimal Swadge mode to be displayed on top of others
+/// Note that it only gets init/exit and mainLoop callbacks.
+static swadgeMode_t* overlayMode = NULL;
```

Comment:
Unless we plan on having multiple different overlays, it'd be a bit cleaner to reference `quickSettingsMode` directly, rather than through this pointer. You'd then need a `bool` which tracks whether the overlay should be shown or not, rather than checking this pointer's null-ness.

Reply:
Ah yeah, that does seem a bit silly doesn't it